### PR TITLE
readmeUpdated

### PR DESCRIPTION
**Pull request by roxor-dj**
Updated readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,68 @@
...
 You may also see any lint errors in the console.
 
-### `npm test`
+#### `npm test`
 
 Launches the test runner in the interactive watch mode.\
 See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-### `npm run format`
+#### `npm run format`
```
Comment (ceilican) [inline README.md:61]: Here too.

File: README.md [hunk 2]
```diff
@@ -1,38 +1,68 @@
...
 
 Formats the code to comply with the `prettier` style.
 
-### `npm run build`
+#### `npm run build`
```
Comment (ceilican) [inline README.md:65]: Here too.

File: README.md [hunk 3]
```diff
@@ -42,7 +72,7 @@ Your app is ready to be deployed!
 
 See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
 
-### `npm run eject`
+#### `npm run eject`
```
Comment (ceilican) [inline README.md:75]: Here too.

File: README.md [hunk 4]
```diff
@@ -52,32 +82,23 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
-
-### Code Splitting
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
 
-### Analyzing the Bundle Size
+## Documentation
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+#### Milkomeda docs
```
Comment (ceilican) [inline README.md:88]: Remove this subsection.

File: README.md [hunk 5]
```diff
@@ -52,32 +82,23 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
+Documentation lives on [**milkomeda docs**](https://docs.milkomeda.com/).
 
-### Making a Progressive Web App
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+#### Djed docs
```
Comment (ceilican) [inline README.md:92]: Let's remove this line.

File: README.md [hunk 6]
```diff
@@ -52,32 +82,23 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
+Documentation lives on [**Djed docs**](https://docs.djed.one/alliance/the-djed-alliance).
```
Comment (ceilican) [inline README.md:93]: Let's move this line to the "What is Djed?" section in the beginning.

File: README.md [hunk 7]
```diff
@@ -1,38 +1,68 @@
-# Getting Started with Create React App
 
+# Djed-Solidity-WebDashboard
+
```
Comment (ceilican) [inline README.md:3]: Could you add some interesting badges/shield in the beginning of the readme?  Have a look here (https://shields.io/) to see how to do it.

File: README.md [hunk 8]
```diff
@@ -1,38 +1,68 @@
...
+Djed is a formally verified crypto-backed autonomous stablecoin protocol. 
+
+It has been researched since Q2 2020, its [whitepaper](https://eprint.iacr.org/2021/1069) has been released in August 2021, and it has multiple [implementations](https://github.com/DjedAlliance) and [deployments](https://docs.djed.one/implementations-and-deployments/deployments). 
+
+Here you can interact with a deployment that uses [these smart contracts](https://github.com/DjedAlliance/Djed-Solidity/commits/Belus) on Milkomeda-C1.
+
+
+## Installation
+
+#### Getting Started with Create React App
```
[the remark this answers is not in the document]
Comment (roxor-dj) [reply README.md:25]: Should I remove here all lines from 22 - 25, or just 25? @ceilican

File: README.md [hunk 9]
```diff
@@ -1,25 +1,51 @@
...
+
+This repository contains the complete source code for a frontend application designed to facilitate user interaction with deployments of the Djed protocol's Solidity implementation. 
+
+With the aim of providing a smooth user experience, the codebase offers a comprehensive set of functionalities for users to engage with Djed deployments efficiently. 
+
+The frontend enables users to explore and interact with various features and capabilities of Djed, leveraging the power of the Solidity implementation. 
+
+By accessing this repository, developers and enthusiasts can delve into the inner workings of the frontend and gain a deeper understanding of how it integrates with Djed's Solidity implementation to enable a robust and user-friendly experience.
+
+Here you can interact with a deployment that uses [these smart contracts](https://github.com/DjedAlliance/Djed-Solidity/commits/Belus) on Milkomeda-C1.
```
Comment (ceilican) [inline README.md:26]: Yes, let's delete this paragraph too.

File: README.md (hunk 7 above)
Comment (ceilican) [reply README.md:2]: OK. Let's leave shields to another PR.

File: README.md [hunk 10]
```diff
@@ -1,25 +1,51 @@
...
+```bash
+npm install
+```
 
-## Available Scripts
+## Usage
 
 In the project directory, you can run:
 
 ### `npm start`
```
Comment (ceilican) [inline README.md:43]: Use triple backquotes (```) instead of a header (###) here.

Comment (ceilican) [reply README.md:43]: Also for "### `npm test`", "### `npm run format`", run build, run eject.

**Review by ceilican — CHANGES_REQUESTED** [review]

**Review by ceilican — APPROVED** [review]

**Pull request merged**